services/tasks-svc/main.go

```diff
@@ -14,14 +17,27 @@ const ServiceName = "tasks-svc"
 var Version string
 
 func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+
 	common.Setup(ServiceName, Version, true)
 
 	eventStore := eventstoredb.SetupEventStoreByEnv()
 	aggregateStore := eventstoredb.NewAggregateStore(eventStore)
 
-	common.StartNewGRPCServer(common.ResolveAddrFromEnv(), func(server *daprd.Server) {
+	echoProjection := echo.NewProjection(eventStore)
```

Comment:
you can't really do anything else than subscribe to a projection, right?
In that case, it's easy to clean up the code by moving the construction into the go func() block